tests/unit/lms/security_test.py

```diff
@@ -82,10 +82,9 @@ def policy(self):
 
 
 @pytest.mark.usefixtures("pyramid_config")
-class TestLTISecurityPolicy:
+class TestLTIUserSecurityPolicy:
     def test_it_returns_empty_identity_if_theres_no_lti_user(self, pyramid_request):
-        pyramid_request.lti_user = None
-        policy = LTISecurityPolicy()
+        policy = LTIUserSecurityPolicy(Mock(return_value=None))
```

Comment:
Can we autospec this so that it ensures that `_get_lti_user()` is called with the right number of args? 

```suggestion
        policy = LTIUserSecurityPolicy(create_autospec(_get_lti_user, return_value=None))
```

Or if you don't want to use the `_get_lti_user()` function specifically (since in future that'll be split into more than one function) then you can do `create_autospace(lambda request: None, return_value=None)` or `security.py` could define an explicit interface for the getter functions and the tests could autospec the interface.

And the same for the other calls to `Mock()` in this PR. In general we should always avoid creating unspecced mocks because you can call any method on them with any arguments and it'll always succeed.